CODEOWNERS

```diff
@@ -29,64 +29,74 @@ Jenkinsfile  @openvinotoolkit/openvino-admins
 
 # OpenVINO Core:
 /src/inference/  @openvinotoolkit/openvino-ie-maintainers
+/cmake/ @openvinotoolkit/openvino-ie-maintainers
+/src/cmake/ @openvinotoolkit/openvino-ie-maintainers
+CMakeLists.txt @openvinotoolkit/openvino-ie-maintainers
+*cmake @openvinotoolkit/openvino-ie-maintainers
+/src/common/cc/ @openvinotoolkit/openvino-ie-maintainers
+/src/common/itt/ @openvinotoolkit/openvino-ie-maintainers
+/src/common/legacy/ @openvinotoolkit/openvino-ie-maintainers
+/src/common/preprocessing/ @openvinotoolkit/openvino-ie-maintainers
+/src/common/util/ @openvinotoolkit/openvino-ie-maintainers
+/thirdparty/ @openvinotoolkit/openvino-ie-maintainers
+/scripts/ @openvinotoolkit/openvino-ie-maintainers
+
 /src/bindings/python/ @openvinotoolkit/openvino-ie-python-api-maintainers
-/src/common/transformations/  @openvinotoolkit/openvino-ie-transformations-maintainers
-/src/common/legacy/  @openvinotoolkit/openvino-ngraph-maintainers
-/src/common/  @openvinotoolkit/openvino-ie-maintainers
+/src/bindings/c/ @openvinotoolkit/openvino-c-api-maintainers
+/src/common/*transformations/  @openvinotoolkit/openvino-ie-transformations-maintainers
 /src/core/  @openvinotoolkit/openvino-ngraph-maintainers
-/src/frontends/  @openvinotoolkit/openvino-ngraph-maintainers
-/src/tests_deprecated/readers/  @openvinotoolkit/openvino-ngraph-maintainers
+
 
 # OpenVINO CPU:
 /src/plugins/intel_cpu/  @openvinotoolkit/openvino-ie-cpu-maintainers @openvinotoolkit/openvino-ie-cpu-developers
-/src/plugins/intel_cpu/thirdparty/onednn/  @openvinotoolkit/openvino-ie-cpu-maintainers @openvinotoolkit/openvino-ie-cpu-developers
+/src/common/snippets/ @openvinotoolkit/openvino-ie-cpu-maintainers
+/src/tests/**cpu** @openvinotoolkit/openvino-ie-cpu-maintainers
 
 # OpenVINO LPT
 /src/common/low_precision_transformations/  @openvinotoolkit/openvino-ie-lpt-maintainers
 
 # OpenVINO GPU:
-/src/inference/include/ie/gpu/  @openvinotoolkit/openvino-ie-gpu-maintainers @openvinotoolkit/openvino-ie-gpu-developers
-/src/inference/include/ie/cldnn/  @openvinotoolkit/openvino-ie-gpu-maintainers @openvinotoolkit/openvino-ie-gpu-developers
```

Comment:
Why did you remove these paths?

Reply:
These paths are not listed in the reference PPT - that's why I removed them.

Reply:
But we have these paths at the current moment. I don't want to change this logic. It is a legacy gpu part

Reply:
Reverted this change.